docs/modules/about/pages/welcome.adoc

```diff
@@ -1,54 +1,40 @@
-= ALEC Documentation
 :imagesdir: ../assets/images
 
-== What is ALEC?
+= About ALEC
 
-The Architecture for Learning Enabled Correlation (ALEC) is a framework for logically grouping related faults (alarms) into higher level objects (situations) with OpenNMS.
+The Architecture for Learning Enabled Correlation (ALEC) provides a machine learning-powered framework for logically grouping related alarms into higher-level situations.
+As the correlation engines learn from users' network environments and their feedback, they will provide more accurate groupings and reduce the total number of notifications generated.
 
-== How does it work?
-
-=== Inventory in OpenNMS is converted to a graph
-
-Nodes, their components and their relations are used to build a graph.
-
-image::model.png[Model example,600]
+Because correlated alarms are likely to have been generated by the same underlying issue, alarm groupings provide a starting point for root cause analysis.
 
-The layout of the graph, and the different types of elements on the graph are what compose "the meta-model".
-In ALEC, the vertices on the graph are referred to as "inventory objects" and edges are used to represent "relationships".
-
-=== Alarms are attached to the graph
-
-Alarms are enriched to help identify which component in the model they relate to.
-The alarms are attached to the graph when they are triggered.
-
-image::model_with_alarms.png[Model example with alarms,600]
+== How does it work?
 
-In OpenNMS, we can now populate the managed object (MO) type and MO instance of an alarm when it is created.
-These fields are used (and required) to relate the alarm to a specific IO on the graph.
+First, your provisioned network inventory is analyzed and converted to a graphical representation.
```

Comment:
Could we say what does the analysis?
```suggestion
First, X analyzes your provisioned network inventory and converts it to a graphical representation.
```

Reply:
Once this PR is merged, I'll cross-reference this sentence with the updated cluster.adoc page.

```suggestion
First, the cluster engine analyzes your provisioned network inventory and converts it to a graphical representation.
```